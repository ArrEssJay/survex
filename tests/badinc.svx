; pos=fail
; include of a nonexistent file
*include nonexistent_file
