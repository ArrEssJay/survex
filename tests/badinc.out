./badinc.svx:2: Couldn't open data file "nonexistent_file"
cavern: No survey data
